components/kyma-environment-broker/internal/broker/broker.go

```diff
@@ -305,6 +301,42 @@ func (b *KymaEnvBroker) LastOperation(ctx context.Context, instanceID string, de
 	}, nil
 }
 
+func (b *KymaEnvBroker) handleDashboardURL(instance *internal.Instance) domain.LastOperationState {
+	b.Dumper.Dump("Get dashboard url for instance ID: ", instance.InstanceID)
+
```

Comment:
maybe we could check if there is a dashboard url in the instance, if not - then ask director for it